src/components/create_client/EditMobileClientBaseClass.js

```diff
@@ -23,6 +27,14 @@ class EditMobileClientBaseClass extends Component {
       }
     };
     this.app = new MobileApp({ ...this.props.ui.app });
+    this.handleTextInputChange = this.handleTextInputChange.bind(this);
+  }
+
+  handleTextInputChange = (valueInput) => {
+    if (this._validate(CREATE_CLIENT_NAME) == 'success') {
+      this.setState({ valueInput, isValid: true });
+    }
+    this.setState({ valueInput, isValid: false });
   }
 
   _validate(propertyName) {
```

Comment:
@dlabaj @jessiehuff getting stuck at the line below regarding the "getProperty" method which exists in `mobileapp.js` it calls `this.getSpec()` and thats where its returning undefined.